### PR TITLE
ci: Auto update firebase api client

### LINKED ISSUE #1420: Update Google Client API automatically
**Author the user story for this feature**

As a Developer, I want to update Google Client API automatically so I will be always up to date with the latest features.

**Describe the solution**
Create scheduled GitHub action which runs client update and create a pull request

**Additional context**
Run it every Monday like dependencies update.
Update documentation

### PR DESCRIPTION
**Pull request by piotradamczyk5**
Fixes #1420

Every Monday morning together with dependencies update ( but on separate PR) pull request with google client library update is created.

Under the hood it use commands (python 2.7 required)
```yaml
pip install google-apis-client-generator
flankScripts shell firebase updateApiJson
flankScripts shell firebase generateJavaClient
```
so you could test this PR locally with them

## Checklist

- [x] Fixed issues with update client
- [x] Create Github Action to update client

### CODE REVIEW

File: flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/GenerateJavaClientCommand.kt [hunk 1]
```diff
@@ -14,9 +14,10 @@ object GenerateJavaClientCommand : CliktCommand(name = "generateJavaClient", hel
     override fun run() {
         checkIfPipInstalled()
         installClientGeneratorIfNeeded()
-        val apiPath = Paths.get("test_api").toString()
+        val firebaseApiPath = Paths.get("", "firebase_apis").toString()
```
Comment (adamfilipow92) [inline flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/GenerateJavaClientCommand.kt:17]: There is any difference between paths from ```Paths.get("", "firebase_apis").toString()``` and ```Paths.get("firebase_apis").toString()```?

**Review by Sloox — COMMENTED** [review]
Are you testing the github actions on a fork?

**Comment by piotradamczyk5** [comment]
> Are you testing the github actions on a fork?

Yes https://github.com/piotradamczyk5/flank/actions/runs/440668950
it fails, because I do not have custom scope token on it

File: flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/GenerateJavaClientCommand.kt (hunk 1 above)
Comment (piotradamczyk5) [reply flank-scripts/src/main/kotlin/flank/scripts/shell/firebase/GenerateJavaClientCommand.kt:17]: no, there is not, fixed

**Review by adamfilipow92 — APPROVED** [review]

**Review by pawelpasterz — APPROVED** [review]

**Pull request merged**